scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Rather than writing new code here, any chance this could re-use more of what we already have in utils_cv? We want the code between the 01 and 02 notebooks to be as similar as possible since this 02 notebook also serves as a proof that the implementation in the 01 notebook are correct</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Evaluate() is spited into several functions so that we don't have to change parameters in evaluate(). And reuse sub-functions in evaluate().